src/main/java/com/deque/axe/android/AxeRuleResult.java

```diff
@@ -40,21 +41,29 @@
    */
   public final String ruleSummary;
 
+  /**
+   * The impact of the rule [0=MINOR, 1=MODERATE, 2=SERIOUS, 3=CRITICAL, 4=BLOCKER].
+   */
+  public @AxeImpact int impact;
```

Comment:
Enums are fine now, it'd be preferable for me because then I can reference them from the library.